lms/static/js/student_account/views/FormView.js

```diff
@@ -172,7 +172,7 @@
 
                         if (key) {
                             test = this.validate(elements[i]);
-                            if (test.isValid) {
+                            if (test.isValid || $el.attr('id') === 'login-email') {
```

Comment:
I understand -- but we can't do that. At least, this would never get through upstream review. The best you could do about this is make it configurable, but no way that edX reviewers would accept straight up disabling error state for the email field.